packages/lesswrong/components/common/TabNavigationMenu/SubforumsList.tsx

```diff
@@ -38,11 +39,13 @@ const INITIAL_LIMIT = 3
 
 const SubforumsList = ({ onClick, classes }) => {
   const { results } = useMulti({
-    terms: {view: 'currentUserSubforums'},
+    // TODO possibly make it include subscribed tags
+    terms: {view: 'coreTags'},
```

Comment:
This is the thing I'm least sure of. I think people might be annoyed that they have joined subforums which now won't appear in the sidebar, but I don't want to do too heavy a redesign at this point